### ISSUE TITLE
Hosting compiled scripts

### ISSUE THREAD

**Issue by hoppfrosch** [ahkscript/awesome-AutoHotkey#89]
As a followup of #88:

As a solution to avoid dead links on source or exe, @hi5 suggested:
> host such compiled scripts as "releases" of this repo or a new for this purpose setup repo (Awesome-AutoHotkey-binaries) :question:  that way the link(s) will always work unless GH goes bankrupt in which case we have other problems

What do you think about this idea?

**Comment by hoppfrosch** [comment]
I'm ambivalent about this:

Pro: As @hi5 said, the link will always work - we may not loose any things we referenced to ...

Contra:
* Copyright issues - are we allowed to simple copy ALL the wanted files? Who looks for potential copyright violations from our side?
* Up-to-date Issues: refering to the original location, we are sure to have always the latest verion linked ... Copying a certain version we soon might get outdated by newer version ...

**Comment by hi5** [comment]
I agree regarding copyright etc. I would mostly be useful for material that is truly "lost" for example the Sparrow webserver, it is not available on ahk . net or ahk . com  

The decompiler Gist is a script, so in principle that would run as is anyway so there is not actually a need for hosting a compiled version.

**Comment by joedf** [comment]
[sparrow-0.1.3.zip](https://github.com/ahkscript/awesome-AutoHotkey/files/219680/sparrow-0.1.3.zip)

**Comment by hoppfrosch** [comment]
What about the attached Sparrow-Zip file? Should an entry be generated for this? Please open a new issue for this, providing the needed info (forum thread, historical or actual? ...)

**Comment by hi5** [comment]
Sparrow is already listed in Historical, the link to ge.tt is still working but like the decompiler that might end some day.

**Comment by joedf** [comment]
yeah... all my files from ge.tt, my "AutoHotkey Scripts Archive" was deleted due to a "virus" detected... :dizzy_face:
Anyhow, I have them now here: https://autohotkey.com/joe_df/autohotkey_scripts_archive/
